Formula/cstore_fdw.rb

```diff
@@ -19,7 +19,7 @@ class CstoreFdw < Formula
   end
 
   # https://www.citusdata.com/blog/2021/03/06/citus-10-columnar-compression-for-postgres/
-  deprecate! date: "2021-03-06", because: "cstore_fdw has been integrated into Citus"
+  disable! date: "2022-11-29", because: "cstore_fdw has been integrated into Citus"
```

Comment:
This date is wrong/misleading and was deprecated 9 days ago: https://github.com/Homebrew/homebrew-core/commit/c19ffc19691369e45558783a92326e1f93a6ea6b

Reply:
That is a good call!

Reply:
Weird, I thought my script would check the commit date. Anyway, I removed this one.